packages/react/xr/src/hooks.ts

```diff
@@ -102,3 +104,132 @@ export function useSessionModeSupported(mode: XRSessionMode, onError?: (error: a
 export function useSessionFeatureEnabled(feature: string) {
   return useXR(({ session }) => session?.enabledFeatures?.includes(feature) ?? false)
 }
+
+export interface ControllerLocomotionTranslationOptions {
+  speed?: number
+}
+export interface ControllerLocomotionRotationOptions {
```

Comment:
a imo cleaner options variant would be:

```
export type ControllerLocomotionRotationOptions {
viewControlDeadZone?: number
} & ({type?: "snap", degree?: number } | {.type: "smooth", speed?: number  })
```

then it would be also be great if you could document the properties (e.g. `viewControlDeadZone` I have no clue what it is exactly)

Reply:
I have them all documented in the `all-hooks.md` file, but if you mean through jsdoc, then I'll add that as well 👍